jersey/src/main/java/com/netflix/conductor/server/resources/MetadataResource.java

```diff
@@ -81,6 +81,13 @@ public WorkflowDef get(@PathParam("name") String name, @QueryParam("version") In
 	public List<WorkflowDef> getAll() throws Exception {
 		return service.getWorkflowDefs();
 	}
+
+	@DELETE
+	@Path("/workflow/{name}")
+	@ApiOperation("Removes workflow definition")
+	public void unregisterWorkflowDef(@PathParam("name") String name, @QueryParam("version") Integer version) throws Exception {
```

Comment:
So version is optional, right? If not specified, version might be 0 by default and MetadataService would return an exception `Version is not valid`. If we do not want to make version optional, can we add it to Path? If it can be optional, does it deletes the latest version by default?

Reply:
It is required as per now. I can add to the path.